docs/hardware.md

```diff
@@ -1,8 +1,18 @@
 Hardware requirements
 ---
-* Disk space: 500 GB (400GB for Bitcoin blockchain + some room)
-  * Bitcoin Core pruning is not supported at the moment because it's not supported by c-lightning. It's possible to use pruning but you need to know what you're doing.
-* RAM: 2GB of memory. ECC memory is better. Additionally, it's recommended to use DDR4 memory with targeted row refresh (TRR) enabled (https://rambleed.com/).
+* RAM: 2GB. ECC memory is better. Additionally, it's recommended to use DDR4 memory with
+  targeted row refresh (TRR) enabled (https://rambleed.com/).
+* Disk space: 500 GB (400GB for Bitcoin blockchain + some room) for an unpruned
+  instance of Bitcoin Core.
+  * This can be significantly lowered by enabling pruning.
+    Note: Pruning is not supported by `lnd` and `electrs`.
```

Comment:
This doesn't seem to be true anymore for [lnd](https://github.com/lightningnetwork/lnd/blob/master/docs/INSTALL.md#available-backend-operating-modes)